#437 - Added documentation for specify and no-ring 0 specify

Mr-KayZ opened the pull request:
LMK if this is works!

TheKrol commented:
Need to update the no-ring download to the correct one. I'm not sure if there is a link for it right now, just might need to get it off github.

Mr-KayZ commented:
Added the following changes:
- Changed specify section in Making a System Dossier - Added links for both Specify articles (normal and no ring 0)
- Changed up wording of both specify and no ring 0 articles with active feedback (thanks again xscissors, Krol and Pipe!)

PipeItToDevNull commented:
Combining the callouts is the last thing I see, maybe like below 

> No sensitive information is included in the report, we will not ask you to share private information. You can vew an example specify report here.
> 
> More information can be found on our README on GitHub.

And I don't think we need the ! on callouts https://rtech.support/meta/contributing/#alerts

Mr-KayZ commented:
> And I don't think we need the ! on callouts https://rtech.support/meta/contributing/#alerts

Nope we definitely need those: 
![image](https://github.com/user-attachments/assets/b1ab36bc-c2f1-4dbc-8830-95df4286facb)

PipeItToDevNull commented:
> > And I don't think we need the ! on callouts https://rtech.support/meta/contributing/#alerts
> 
> 
> 
> Nope we definitely need those: 
> 
> ![image](https://github.com/user-attachments/assets/b1ab36bc-c2f1-4dbc-8830-95df4286facb)
> 
> 
> 
> As for the rest, done for both no-ring and default

I meant on the end "Important!"
